### DISCUSSION TITLE
页面未找到，不是很pro

### DISCUSSION THREAD

**Discussion by KashingLiu** [ant-design/ant-design-pro#11099] [General]
如题，https://pro.ant.design/zh-CN/docs/request 链接中的第一个外链就提示页面不存在。
https://umijs.org/zh-CN/plugins/plugin-request#request
<img width="1336" alt="image" src="https://github.com/ant-design/ant-design/assets/22939122/73404705-13c6-4980-82cb-f0356d0ccf71">

**Comment by XueMeijing** [comment]
https://umijs.org/docs/max/request#request 应该是改成这个了，antd文档的引用没更新，上面说umi@4已经弃用改为axios了
 
顺便说一下，如果你是用umi，开发是快了一点，但是要做好踩很多坑的准备
